### PR TITLE
Rename RossumPython to TxScript

### PR DESCRIPTION
**Pull request by pasky**

### CODE REVIEW

File: docs/learn/rossum-formulas/index.md [hunk 1]
```diff
@@ -149,23 +151,23 @@ Substitute is an alias for [`re.sub`](https://docs.python.org/3/library/re.html#
 #### Formula field
 
 ```py
-substitute(r"[^0-9]", r"", field.document_id)  # Remove non-digit characters
+substitute(t"[^0-9]", t"", field.document_id)  # Remove non-digit characters
```
Comment (mrtnzlml) [inline docs/learn/rossum-formulas/index.md:154]:
```suggestion
substitute(r"[^0-9]", r"", field.document_id)  # Remove non-digit characters
```

File: docs/learn/rossum-formulas/index.md [hunk 2]
```diff
@@ -149,23 +151,23 @@ Substitute is an alias for [`re.sub`](https://docs.python.org/3/library/re.html#
...
 ```
 
 Could also be written as (`re` is imported automatically):
 
 ```py
-re.sub(r"[^0-9]", r"", field.document_id)
+re.sub(t"[^0-9]", t"", field.document_id)
```
Comment (mrtnzlml) [inline docs/learn/rossum-formulas/index.md:160]:
```suggestion
re.sub(r"[^0-9]", r"", field.document_id)
```

File: docs/learn/rossum-formulas/index.md [hunk 3]
```diff
@@ -149,23 +151,23 @@ Substitute is an alias for [`re.sub`](https://docs.python.org/3/library/re.html#
...
 #### Serverless function
 
 ```py
-from rossum_python import RossumPython, substitute
+from txscript import TxScript, substitute
 
 # …
 
-substitute(r"[^0-9]", r"", field.document_id)
+substitute(t"[^0-9]", t"", field.document_id)
```
Comment (mrtnzlml) [inline docs/learn/rossum-formulas/index.md:170]:
```suggestion
substitute(r"[^0-9]", r"", field.document_id)
```

**Review by mrtnzlml — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. There seem to be some unwanted (?) changes. Could you please have a look?

**Review by mrtnzlml — APPROVED** [review]

**Comment by pasky** [comment]
Thanks for catching these, sorry about that!

**Pull request merged**